platform/branding-api/src/main/java/org/codice/ddf/branding/BrandingRegistryImpl.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+
+package org.codice.ddf.branding;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class BrandingRegistryImpl implements BrandingRegistry {
+
+    private List<BrandingPlugin> brandingPlugins = Collections.emptyList();
+
+    /**
+     * This method returns the name portion of the name & version of the product, as defined
+     * by the highest priority {@link BrandingPlugin} in the system..
+     * E.g, if the full name is "DDF v1.0.0", this will return "DDF"
+     *
+     * @return a String representing the name of the product
+     */
+    @Override
+    public String getProductName() {
+        return !getBrandingPlugins().isEmpty() ?
```

Comment:
I used the getter mainly for unit testing. I couldn't figure out how to mock a method that takes a method reference (see the `LandingPageTest`) so instead I had to instead mock out the `BrandingPlugins` instead. I think it does lead to cleaner code though